02-basics.Rmd

```diff
@@ -132,13 +132,13 @@ ncol(crashes)
 The results of the previous commands tell us that the dataset has 3 rows and 2 columns.
 We will use larger datasets, with thousands of rows and tens of columns, in later chapters but for now it's good to 'start small' to understand the basics of R.
 
-## Subsetting by index or name
+## Subsetting by index or name CHANGE TITLE OR BE CLEARER WHEN YOU ARE SUBSETTING BY INDEX OR NAME IN THE TEXT BELOW
 
 Subsetting returns part of an R object. 
-It can be done by providing numbers representing the positions of the elements we want (e.g. the 2^nd^ element) or with a logical vector, with values associated with `TRUE` returned. 
+It can be done by providing numbers representing the positions of the elements we want (e.g. the 2^nd^ element) or with a logical vector *NEED TO EXPLAIN VECTOR, with values associated with `TRUE` or `FALSE` returned. 
```

Comment:
Good point. How about:

```
The most basic type of R object is a *vector*, a sequence of values of the same type.
`x`, `y` and `z` were all *numeric vectors* with a length of 1; `casualty_type` is a *character vector* (because it contains letters that cannot be added) or length 3; `casualty_age` is a *numeric vector* of length three.
In R, subsets of vectors can be returned by providing numbers representing the positions of the elements we want (e.g. '2' representing selection of the 2^nd^ element) or with logical (`TRUE` or `FALSE`) values associated with the elements that should be selected. 
```

Reply:
Few more edits below- hope they make sense 


The most basic type of R object is a *vector*, a sequence of values of the same type, FOR EXAMPLE, TYPE COULD BE NUMERIC.
`x`, `y` and `z` were all *numeric vectors* with a length of 1; `casualty_type` is a *character vector* (because it contains letters that cannot be added) OF length 3; `casualty_age` is a *numeric vector* of length 3. CHANGED THREE TO 3 FOR COSNISTENCY
In R, subsets of vectors can be returned by providing numbers representing the positions of the elements we want WITHIN THE VECTOR (e.g. '2' representing selection of the 2^nd^ element) or with logical (`TRUE` or `FALSE`) values associated with the elements that should be selected.

Reply:
SUGGESTED WORDING CHANGES BELOW

The most basic type of R object is a *vector*, a sequence of values of the same type FOR EXAMPLE, A SET OF NUMBER.
`x`, `y` and `z` were all *numeric vectors* with a length of 1; `casualty_type` is a *character vector* (because it contains letters that cannot be added) OF length 3; `casualty_age` is a *numeric vector* of length 3(CHANGED TO THREE FOR CONSISTENCY).


In R, subsets of vectors can be returned by providing numbers representing the positions of the elements IN THE VECTOR  we want (e.g. '2' representing selection of the 2^nd^ element) or with logical (`TRUE` or `FALSE`) values associated with the elements that should be selected.